app/src/androidTest/java/ca/dal/cs/athletemonitor/athletemonitor/MainActivityTest.java

```diff
@@ -29,7 +29,9 @@
 
     @Before
     public void loginToTestUser() {
+        // Authenticate user and set login state to false so Firebase does not keep testuser online
         AccountManager.authenticate(new User("testuser", "abc"), null);
```

Comment:
This isn't guaranteed to work due to asynchronous calls, the setUserLoginState should be called from the authenticate callback, but it is minor issue that will be resolved tomorrow by #11 .

Reply:
Right, makes sense, we can just remove this extra step once that issue is fixed.